ndk/sample/build.gradle.kts

```diff
@@ -60,6 +60,16 @@ android {
                 ),
             )
         }
+        getByName("debug") {
+            isMinifyEnabled = false
+            signingConfig = signingConfigs.getByName("debug")
+            addManifestPlaceholders(
+                mapOf(
+                    "sentryDebug" to true,
+                    "sentryEnvironment" to "debug",
+                ),
+            )
+        }
```

Comment:
Any specific reason why we need this change? I don't see `sentryDebug` / `sentryEnvironment` being used, so I'd simply remove this block.

Reply:
Good catch, this was left over from the CI testing we did (but is unnecessary to remain within this PR).